mattermost/mattermost-webapp
# [MM-22212] Add channel scheme websocket event

#### Summary
Adds a websocket event to dispatch `getMyChannelMember` when a channel_scheme is either created or deleted to update the list of roles tied to that channel.

Related Server PR: https://github.com/mattermost/mattermost-server/pull/14057

Test steps: 
1.  Login to a channel as regular member
2. Login as sysadmin in different tab and make the channel read only
3. Ensure that the channel changes to read only without reload for the regular member
